scalalib/test/src/giter8/Giter8Tests.scala

```diff
@@ -0,0 +1,33 @@
+package mill.scalalib.giter8
+
+import mill.T
+import mill.util.{TestEvaluator, TestUtil}
+import utest._
+import mill.main.MainModule
+import mill.api.Loose.Agg
+
+object Giter8Tests extends TestSuite {
+  
+  object mainModule extends TestUtil.BaseModule with MainModule
+
+  override def tests: Tests = Tests {
+    test("init") {
+      val eval = new TestEvaluator(mainModule)
+      test("hello world") {
+        val outDir = os.temp.dir(prefix = "mill_giter8_test")
+        val giter8Args = Seq(
+          "sake92/mill-scala-hello.g8",
```

Comment:
I think, for the test we should use a local test template. G8 should support that. My intend it to avoid dependencies to external (mutable) targets and reduce avoidable network traffic. Also, that way we can make the test case as simple or complex as we want, regardless of its usefulness for others.

Reply:
Yep, makes sense.